src/commands/generate/typeGenerator.ts

```diff
@@ -267,7 +267,10 @@ export async function generateUnion(
         docs: definition.docs != null ? [{ description: definition.docs }] : undefined,
         isExported: true,
         name: unionTsType,
-        type: unionSourceFileInput.memberInterfaces.map(iface => iface.name).join(" | "),
+        type:
+            unionSourceFileInput.memberInterfaces.length === 0
+                ? "any"
```

Comment:
Leaning towards `unknown` here if possible since it has better type safety guarantees than `any`.

```suggestion
                ? "unknown"
```

https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type